lib/readline.js

```diff
@@ -120,8 +119,8 @@ function Interface(input, output, completer, terminal) {
   this.input = input;
   this.historySize = historySize;
   this.removeHistoryDuplicates = !!removeHistoryDuplicates;
-  this.crlfDelay = Math.max(kMincrlfDelay,
-                            Math.min(kMaxcrlfDelay, crlfDelay >>> 0));
+  this.crlfDelay = crlfDelay === Infinity ?
```

Comment:
being an integer might have been more important at some point in the past, but if you look at the current uses:
https://github.com/Azard/node/blob/d90647d0952091bd7053404945f1e01c57f5e98c/lib/readline.js#L412
and
https://github.com/Azard/node/blob/d90647d0952091bd7053404945f1e01c57f5e98c/lib/readline.js#L925
AFAICT treating it as just a `number` should work fine... (just need to make sure it's not passed to C++ and cast to `int`).